diff --git a/config.go b/config.go
index 5e5b4f4..a2746ef 100644
--- a/config.go
+++ b/config.go
@@ -103,7 +103,7 @@ func DefaultConfig() Config {
 		ApplicationOwner:         "Anonymous",
 		BaseURL:                  "",
 		BodyLimit:                defaultBodyLimitConfig,
-		DataDirectory:            "/usr/share/drasl",
+		DataDirectory:            "@out@/share/drasl",
 		DefaultAdmins:            []string{},
 		DefaultPreferredLanguage: "en",
 		Domain:                   "",
