diff --git a/config.go b/config.go
index d04949b..9f17413 100644
--- a/config.go
+++ b/config.go
@@ -71,7 +71,7 @@ func DefaultConfig() Config {
 	return Config{
 		InstanceName:             "Drasl",
 		StateDirectory:           "/var/lib/drasl",
-		DataDirectory:            "/usr/share/drasl",
+		DataDirectory:            "@out@/usr/share/drasl",
 		ApplicationOwner:         "Unmojang",
 		LogRequests:              true,
 		SignPublicKeys:           false,
